_layouts/participant.html

```diff
@@ -2,6 +2,11 @@
 
 {%- assign tracks = site.data.tracks |where: "year", page.year |first -%}
 {%- assign tracks = tracks.tracks -%}
+{%- if page.sysDescrUrl contains "://" -%}
+    {%- assign sysDescrUrl = page.sysDescrUrl -%}
+{%- else -%}
+    {%- assign sysDescrUrl = "../system-descriptions/" | append: page.sysDescrUrl -%}
```

Comment:
Should we add the `page.year` here, to avoid having everything in the same folder, and not to have to move the 2019 descriptions around now?

Reply:
As the participant is under `2019/participants/name.html`, the relative path points to `2019/sytem-description/name.pdf`.  I'm not sure why an absolute path is better than a relative path here.

Reply:
Ok, then it works.  I think the relative path is good.